infrastructure/kubernetes/common/monitoring/README.md

```diff
@@ -32,6 +45,19 @@ Use `kubectl` to create the secrets or apply changes:
 kubectl create secret generic -n monitoring grafana-secret \
   --from-literal=admin-user=<USER> \
   --from-literal=admin-password=<PASSWORD>
+
+```
+
+And under grafana-auth-google-secret:
+- client-id
+- secret-id
+
+Use `kubectl` to create the secrets or apply changes:
+```Shell
+kubectl create secret generic -n monitoring grafana-secret \
```

Comment:
```suggestion
kubectl create secret generic -n monitoring grafana-auth-google-secret \
```

Reply:
addressed in: fcf0ade5a8d87a56460dba6164d0810cc5cdc65c